index.js

```diff
@@ -72,7 +72,8 @@ function beautify(error, collection, values, messages) {
                 indexes[indexName].forEach(function (field) {
                     var path = field[0];
                     var props = {
-                        type: 'Duplicate value',
+                        type: 'unique',
+                        kind: 'unique',
```

Comment:
Looks like `Mongoose.ValidatorError` only considers the `type` property from the properties object and [internally converts it to a `kind` attribute](https://github.com/Automattic/mongoose/blob/4.11.1/lib/error/validator.js#L30). So I don’t think we need to supply a `kind` attribute.